larq_compute_engine/tflite/kernels/bmaxpool.cc

```diff
@@ -72,6 +73,8 @@ TfLiteStatus Prepare(TfLiteContext* context, TfLiteNode* node) {
 }
 
 TfLiteStatus Eval(TfLiteContext* context, TfLiteNode* node) {
+  ruy::profiler::ScopeLabel label("Binary MaxPool");
```

Comment:
Not sure if this should be like so or `"BMaxPool2D"`, like the existing `"BConv2D"` label for the binary conv op.